libsolidity/AST.h

```diff
@@ -999,6 +1000,7 @@ class FunctionCall: public Expression
 private:
 	ASTPointer<Expression> m_expression;
 	std::vector<ASTPointer<Expression>> m_arguments;
+	std::vector<std::string> m_names;
```

Comment:
Please change this to `std::vector<ASTPointer<ASTString>> m_names;`. Yes, it is ugly, but it should be consistent and allows for some planned optimizations to be made more fluently.